scalyr_agent/scalyr_client.py

```diff
@@ -700,7 +700,7 @@ def __receive_response(self, body_str, send_time):
                 response_as_json = scalyr_util.json_decode(response)
             except Exception:
                 # TODO: Do not just catch Exception.  Do narrower scope.  Also, log error here.
-                log.error(
+                log.info(
```

Comment:
Yeah, for now it's probably better to start with ``warning`` (makes it easier for us to find those log messages).

In case people still find it confusing, we can then change it to info.